olive/passes/onnx/quantization.py

```diff
@@ -332,6 +332,9 @@ def _run_for_config(
             assert (
                 config["dataloader_func"] or config["data_config"]
             ), "dataloader_func or data_config is required for static quantization."
+            # whether to prepare qnn config
```

Comment:
could we move them into validate_search_point?

Reply:
Yeah, I can move it there. Although, I have been thinking that version check in validate_search_point doesn't work well for non local systems. Validate_search_point is run by the engine so the version of the engine machine might be different from the host/target. But we can address that later.

Reply:
Actually, I think it is better inside the run itself. There is only one pass `transformers optimization` which does the version check in search point validation. But that is technically not a good way because of what i described above.

Reply:
could we add comments about the reason that this kind of logic doesn't belong to validate_search_point? actually, I don't know it doesn't work well with non-local system.